rust/processor/src/models/ans_models/ans_utils.rs

```diff
@@ -19,12 +19,12 @@ use serde::{Deserialize, Serialize};
 pub const DOMAIN_LENGTH: usize = 64;
 
 #[derive(Serialize, Deserialize, Debug, Clone)]
-struct OptionalString {
+pub struct OptionalString {
```

Comment:
this looks like it's generic type. let's move it to some generic utils file

Reply:
sounds like a good idea. let's do this separately.